File: python-pr/52/python/code-quality-readability-maintainability/string_handler_03.py
# {fact rule=code-quality-readability-maintainability@v1.0 defects=1}
def handle_string(s):
    return s.replace('a','b').replace('b','c').replace('c','d').replace('d','e').replace('e','f').upper().lower().strip().split(' ')[0] if s else 
# {/fact}

File: js-pr/56/javascript/code-quality-readability-maintainability/string_handler_03.js
// {fact rule=code-quality-readability-maintainability@v1.0 defects=1}
function handleString(s) {
    return s ? s.replace('a','b').replace('b','c').replace('c','d').replace('d','e').replace('e','f').toUpperCase().toLowerCase().trim().split(' ')[0] : "";
}
// {/fact}